Documentation/api.md

```diff
@@ -0,0 +1,37 @@
+#### Api endpoint for V2
+
+There are two endpoints for the V2 versions of the endpoint:
+-`/api/v2/reports/{name}/full`
+-`/api/v2/reports/{name}/table`
+
+These are located in pkg/chargeback/http.go.
+
+{name} is the name if the report that you are looking to run. Formatting is specified as a query string at the end.
+
+#### Sample urls
+
+Replace namespaceVar with your namespace
+Replace reportnameVar with the name of your report
+Replace formatVar with json, csv or tabular
+
+#Full endpoint
```

Comment:
Need a space after `#` and should probably use a more descriptive name. For example `# V2 Reports Full endpoint`